### ISSUE TITLE
Overlapping events

### ISSUE THREAD

**Issue by claudep** [bitmazk/django-calendarium#25]
Currently, there is no way to prevent several events on a single category to overlap. Is it something that might be on a new feature list?

**Comment by Tyrdall** [comment]
@claudep This should be fixed, yes. Our problem is, that we currently don't have any project running this app and so we don't have the time to work on new features. We can support you, though, if you want to improve it.

**Comment by dkaufhold** [comment]
@claudep Just to understand your request: Do you think, that the app should warn the user about events, that take place at the same time or event prevent their creation?

We're still discussing if this should be a feature or not =)

**Comment by claudep** [comment]
Yes, prevent overlapping events. This could be an optional flag on the event category object, not necessarily the default behaviour.

**Comment by dkaufhold** [comment]
I guess, from our side, this would be labeled ``wontfix``. We didn't require such a behaviour yet and basically trusted in the user managing events entirely like other calendars do as well (e.g. Google calendar).

But as Tyrdall mentioned already, we can support you, if you wish to contribute this as an optional feature.

**Comment by Ab3nXY** [comment]
What if we want more than one  event in a given day?

**Comment by BobBowles** [comment]
I came across this problem some years ago when I was looking for a diary and resource scheduling app for my wife's web site. FWIW I ended up writing my own. While inspired by `django-calendarium` the object model design is completely different.
